### PR TITLE
OHM-328: Add fulltruncate query mode for transactions

### PR DESCRIPTION
**Pull request by hnnesv**
OHM-328

### CODE REVIEW

File: src/api/transactions.coffee [hunk 1]
```diff
@@ -40,6 +44,21 @@ getProjectionObject = (filterRepresentation) ->
       return { "request.body": 0, "request.headers": 0, "response.body": 0, "response.headers": 0, orchestrations: 0, routes: 0 }
 
 
+truncateTransactionDetails = (trx) ->
+  truncateSize = apiConf.truncateSize ? 20000
```
Comment (rcrichton) [inline src/api/transactions.coffee]: Maybe default to 15000 as that is what is in the default config.

File: src/api/transactions.coffee [hunk 2]
```diff
@@ -40,6 +44,21 @@ getProjectionObject = (filterRepresentation) ->
...
+
+  trunc = (t) ->
+    if t.request?.body? and t.request.body.length > truncateSize
+      t.request.body = t.request.body[...truncateSize] + apiConf.truncateAppend
```
Comment (rcrichton) [inline src/api/transactions.coffee]: Does truncateAppend need to be defaulted like truncateSize above.

**Comment by rcrichton** [comment]
LGTM, just some minor comments.

File: src/api/transactions.coffee (hunk 2 above)
Comment (hnnesv) [inline src/api/transactions.coffee]: Yeah it does - good catch thanks

**Comment by hnnesv** [comment]
@rcrichton ready for re-review

**Comment by rcrichton** [comment]
LGTM :+1:

**Pull request merged**